master/master.cfg

```diff
@@ -46,32 +46,52 @@ WORKER_SECRET = Path('halide_bb_pass.txt').read_text().strip()
 WEBHOOK_SECRET = Path('webhook_token.txt').read_text().strip()
 WWW_PASSWORD = Path('buildbot_www_pass.txt').read_text().strip()
 
+# SERVER SETTINGS
+
+ARTIFACTS_DIR = os.environ.get('HALIDE_BB_MASTER_ARTIFACTS_DIR', '/home/abadams/artifacts')
+REPO_DIR = Path(__file__, '..', '..').resolve()
+
 # LLVM
 
 # At any given time, we test (at least) 3 LLVM versions:
 # - the current trunk (changes daily)
 # - the most recent release (expected to be stable)
 # - an older release (expected to be stable)
 #
-# the branches that correspond to these will rotate as new versions
-# are released, but the underlying test logic should not need changing
+# The branches that correspond to these will rotate as new versions
+# are released, but the underlying test logic should not need changing.
 
-LLVM_TRUNK_BRANCH = 'main'
-LLVM_RELEASE_BRANCH = 'release/11.x'
-LLVM_OLD_BRANCH = 'release/10.x'
+Version = namedtuple('Version', ['major', 'minor', 'patch'])
+VersionedBranch = namedtuple('VersionedBranch', ['ref', 'version'])
 
-_LLVM_BRANCHES = [LLVM_TRUNK_BRANCH, LLVM_RELEASE_BRANCH, LLVM_OLD_BRANCH]
+LLVM_TRUNK = 'trunk'
```

Comment:
Are these just meant to be unique identifiers? Or user-readable names? Or what? It's not clear at first read. If they are just unique keys, would we be better off with an enum? (I think this bothers me a bit since there are contexts where we identify a branch by these ids, and other contexts where we use the git branch name.)

Reply:
They're meant to be short, user-readable (they appear in builder names), identifiers for the various branches.